FirebaseAppDistribution/Sources/FIRFADLocalStorage.m

```diff
@@ -13,24 +13,22 @@
 // limitations under the License.
 #import "FIRFADLocalStorage+Private.h"
 
+static NSString *const kSignInStateKey = @"FIRFADSignInState";
 @implementation FIRFADLocalStorage
 
-- (instancetype)init {
-  self = [super init];
-
-  return self;
-}
-
-- (BOOL)isTesterSignedIn {
-  return YES;
++ (BOOL)isTesterSignedIn {
+  NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
```

Comment:
FYI, we experienced thread-safety issues with `NSUserDefaults` on some iOS version, so we had to opt out for a custom thread-safe implementation [GULUserDefaults](https://cs.opensource.google/firebase-sdk/firebase-ios-sdk/+/master:GoogleUtilities/UserDefaults/Private/GULUserDefaults.h). It modifies the same user default but via C API under the hood. You may consider using  `GULUserDefaults` if you access it outside of main thread.

Reply:
Good point @maksymmalyhin! We talked about this offline and will likely switch to using GULUserDefaults

Reply:
Done!